secure-enclave/values.yaml

```diff
@@ -2,42 +2,100 @@
+    ##
     provisioner: ebs.csi.aws.com
+    ## @param aws.storageClass.volumeBindingMode this defines the storage class volume binding mode 
+    ##
     volumeBindingMode: WaitForFirstConsumer
+    ## @param aws.storageClass.parameters this defines the storage class parameters 
+    ##
     parameters:
+      ## @param aws.storageClass.parameters.csi.storage.k8s.io/fstype this defines the filesystem type for the storage class 
+      ##
       csi.storage.k8s.io/fstype: xfs
+      ## @param aws.storageClass.parameters.type this defines the type of the storage class 
+      ##
       type: io1
+      ## @param aws.storageClass.parameters.iopsPerGB this defines the IOPS per GB for the storage 
+      ##
       iopsPerGB: "50"
+      ## @param aws.storageClass.parameters.encrypted  this defines if the storage is encrypted 
+      ##
       encrypted: "true"
+    ## @params aws.storageClass.allowedTopologies this defines the allowed toplogies, for eg. the zone where the storage class will be created in.
+    ## 
     allowedTopologies:
       - matchLabelExpressions:
           - key: topology.ebs.csi.aws.com/zone
             values:
               - us-east-1
 
+## @param networkPolicy this defines if the network policy restrictions are enforced.
+## Few limitations: 
+## FQDNs Egress restrictions
+## TODO. Enforce Ingress only to some pods
+##
+networkPolicy:
+  ## @param networkPolicy.enabled this enables or disables the network policy
+  enabled: false
+
+## @param managementApp this defines all the configurations related to the management app
+##
+managementApp: 
+  ## @param managementApp.memberId this defines the id of the member deploying the enclave
+  ##
+  memberId:
+  ## @param managementApp.endpoint this defines the endpoint where the management app is available. 
+  ##
+  endpoint: 
+    ## @param managementApp.endpoint.protocol this defines the protocol used to access the management app 
+    ##
+    protocol: https
+    ## @param managementApp.endpoint.host this defines the host where the management app is available
+    ## This value is required
+    host: 
+    ## @param managementApp.endpoint.port this defines the port where the management app is available
+    ##
+    port: 443
+  ## @param managementApp.privateKey this defines the private key of the member deploying the enclave
+  ##
+  privateKey: |
+      -----BEGIN PRIVATE KEY-----
+      -----END PRIVATE KEY-----
+    
+
+## @param setupApp this defines the setup app configuration
+##
 setupApp:
+  ## @param setupApp.enabled this defines if the setup app should be deployed
+  ##
   enabled: true
+  ## @param setupApp.name this defines the name of the deployment and containers for the setup app
+  ##
   name: setup-app
+  ## @param setupApp.roles this defines the roles needed by the setup app service account to manage jobs, secrets and deployment for the research container
+  ##
   role:
     rules:
```

Comment:
These are too permissive, but I think that is OK for now as we test. We do not want them to be in the values files because we do not want the user to change them, but lets leave them as is and where is for now; add a to do comment to harden and move out of values later